ui-modules/utils/catalog-deleter/catalog-deleter.less

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+.brooklyn-catalog-deleter {
+}
```

Comment:
Unused.

Reply:
i kinda like to have empty css rules for classes on directives -- even if they don't have any style yet there is a clear place it can be added.  i wouldn't go out of my way to make it but in this case i copied `updater` and so it was already there, seemed better to keep it empty than remove it.